spec/features/organisation_spec.rb

```diff
@@ -136,7 +136,7 @@
       end
 
       it "shows a create button for that organisation" do
-        expect(page).to have_button("Create a new lettings log for this organisation")
```

Comment:
How come this is changing?

Reply:
We introduced a new button to "View {lettings/sales} bulk uploads" within the organisation page. This button's text was shortened in the designs to comfortably fit in the new button.